src/main/java/teammates/ui/webapi/CreateFeedbackQuestionAction.java

```diff
@@ -26,9 +36,17 @@ void checkSpecificAccessControl() throws UnauthorizedAccessException {
         String courseId = getNonNullRequestParamValue(Const.ParamsNames.COURSE_ID);
         String feedbackSessionName = getNonNullRequestParamValue(Const.ParamsNames.FEEDBACK_SESSION_NAME);
         InstructorAttributes instructorDetailForCourse = logic.getInstructorForGoogleId(courseId, userInfo.getId());
+        if (instructorDetailForCourse != null) {
+            gateKeeper.verifyAccessible(instructorDetailForCourse,
+                    getNonNullFeedbackSession(feedbackSessionName, courseId),
+                    Const.InstructorPermissions.CAN_MODIFY_SESSION);
+            return;
+        }
```

Comment:
I believe we need to use the isCourseMigrated method here.

There's a case that's not considered here. After migration, datastore entity is still non-null. However, source of truth is now the sql entity, not the datastore one.